models/model.py

```diff
@@ -118,3 +121,66 @@ def forward(self, inputs, attention_mask=None):
         cls_token_state = hidden_states[1]
 
         return cls_token_state
+
+class Net(torch.nn.Module):
```

Comment:
This name is too generic if this file is not purely for GNNs

Reply:
Changed it